tests/behat/pages/page_base.php

```diff
@@ -181,28 +181,28 @@ protected function allocatable_identifier_hash($allocatable) {
     }
 
     /**
-     * @param string $field_name
+     * @param string $fieldname
      * @param int $timestamp
      * @throws \Behat\Mink\Exception\ElementNotFoundException
      */
-    protected function fill_in_date_field($field_name, $timestamp) {
+    protected function fill_in_date_field($fieldname, $timestamp) {
         // Select the date from the dropdown
-        $minute_dropdown_selector = "id_{$field_name}_minute";
-        $hour_dropdown_selector = "id_{$field_name}_hour";
-        $day_dropdown_selector = "id_{$field_name}_day";
-        $month_dropdown_selector = "id_{$field_name}_month";
-        $year_dropdown_selector = "id_{$field_name}_year";
+        $minutedropdownselector = "id{$fieldname}minute";
+        $hourdropdownselector = "id{$fieldname}hour";
+        $daydropdownselector = "id{$fieldname}day";
+        $monthdropdownselector = "id{$fieldname}month";
+        $yeardropdownselector = "id{$fieldname}year";
```

Comment:
I have a feeling these are css ids it would fail to match as a result of this

Reply:
Agreed. Reinstated in commit 55b8a52